### PR TITLE
Cypress: enable Test Isolation

### PR DESCRIPTION
**Pull request by alfonsomthd**
https://docs.cypress.io/app/core-concepts/test-isolation

Now after each test the session is cleared, hence the following refactoring:
- Login occurs in global `beforeEach` hook (and in `before` hooks where is required).
- Removed Logout as no longer necessary (and it's not part of any plugin).

### CODE REVIEW

File: cypress/support.ts [hunk 1]
```diff
@@ -4,18 +4,26 @@ import {
...
+  disableIrrelevantLogging(); // Required as 'before' actions precede 'beforeEach' actions.
 });
 
 beforeEach(() => {
-  // Disable Cypress's default behavior of logging all XMLHttpRequests and fetches.
-  cy.intercept({ resourceType: /xhr|fetch/ }, { log: false });
+  disableIrrelevantLogging();
+  cy.login();
+  cy.visit('/');
+  cy.install();
```
Comment (GowthamShanmugam) [inline cypress/support.ts:26]: if there is not logout, do we really need these three in beforeEach? is this can be moved to before?

Comment (alfonsomthd) [reply cypress/support.ts:26]: No, after each test all is cleared, so we have to perform these actions always. bear in mind that session is cached so it's really performed 1 time.

Comment (alfonsomthd) [reply cypress/support.ts:26]: The test isolation ensures that any test could be run isolated.

**Pull request merged**